questions.md

```diff
@@ -0,0 +1,4 @@
+
+
+1) how does git "remember" current context ?
+2) Are pull request approved by repo manager or quorum or other?
```

Comment:
You have a number of options on GitHub to choose who approves Pull Requests. Ultimately, any collaborators who have write access can approve a Pull Request. You can enable "required reviews:" https://help.github.com/en/articles/enabling-required-reviews-for-pull-requests or even assign CODEOWNERS where a team or individual must approve Pull Requests: https://help.github.com/en/articles/about-code-owners